Cannot use plugin on Linux

On Linux, I get the following error when running a gradle build with the grpc-kotlin plugin:

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':protobufToolsLocator_grpckotlin'.
> Could not find grpc-kotlin-gen-linux-x86_64.exe (io.rouz:grpc-kotlin-gen:0.0.1).
  Searched in the following locations:
      https://nexus.example.com/repository/maven-public/io/rouz/grpc-kotlin-gen/0.0.1/grpc-kotlin-gen-0.0.1-linux-x86_64.exe
```

--- a/README.md
+++ b/README.md
@@ -74,6 +74,9 @@ Add the `grpc-kotlin-gen` plugin to your `protobuf-maven-plugin` configuration (
 Add the `grpc-kotlin-gen` plugin to the plugins section of `protobuf-gradle-plugin`
 
 ```gradle
+def protobufVersion = '3.5.1-1'
+def grpcVersion = '1.13.1'
+
 protobuf {
     protoc {
         // The artifact spec for the Protobuf Compiler
@@ -84,11 +87,11 @@ protobuf {
             artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}"
         }
         grpckotlin {
-            artifact = "io.rouz:grpc-kotlin-gen:0.0.1"
+            artifact = "io.rouz:grpc-kotlin-gen:0.0.2:jdk8@jar"
         }
     }
     generateProtoTasks {
-        ofSourceSet("main")*.plugins {
+        all()*.plugins {
             grpc {}
             grpckotlin {}
         }

--- a/grpc-kotlin-gen/pom.xml
+++ b/grpc-kotlin-gen/pom.xml
@@ -23,4 +23,27 @@
 
     <!-- scope : test -->
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <version>1.5.8.RELEASE</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>repackage</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <mainClass>io.rouz.grpc.kotlin.GrpcKotlinGenerator</mainClass>
+          <layout>JAR</layout>
+          <classifier>jdk8</classifier>
+          <executable>true</executable>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
 </project>

